docs/examples/schema_extra_callable.py

```diff
@@ -8,8 +8,10 @@ class Person(BaseModel):
 
     class Config:
         @staticmethod
-        def schema_extra(schema: Dict[str, Any]) -> None:
-            for prop in schema.get('properties', {}).values():
-                prop.pop('title', None)
+        def schema_extra(
+            schema: Dict[str, Any], model_class: Type["Person"]
```

Comment:
```suggestion
            schema: Dict[str, Any], model_class: Type['Person']
```
Also, are you sure this can't go on one line? Examples have a line length maximum of 80.